jcl/src/java.base/share/classes/com/ibm/tools/attach/target/IPC.java

```diff
@@ -67,6 +68,17 @@
 	static final int LOGGING_UNKNOWN = 0;
 	static final int LOGGING_DISABLED = 1;
 	static final int LOGGING_ENABLED = 2;
+	/**
+	 * Prefixes and names for Diagnostics properties
+	 */
+	public static final String PROPERTY_DIAGNOSTICS_STATUS_PREFIX =
+			"org.openj9.diagnostics.status."; //$NON-NLS-1$
```

Comment:
Just to shorten things up, I think the "org" and the "status" parts can be removed. Seems this could use the OPENJ9_DIAGNOSTICS_PREFIX defined elsewhere.